### PR TITLE
KP, KI, KD setters

### PR DESCRIPTION
**Pull request by eucalvo**

### CODE REVIEW

File: src/UIState/MainMenu.h [hunk 1]
```diff
@@ -32,7 +32,9 @@ class MainMenu : public UIState {
     SET_CALIBRATION_CLEAR,
     SET_TEMP_CALIBRATION,
     SET_PID_AUTO_TUNE,
-    SET_PID_MANUAL_TUNE,
+    SET_KP,
```
Comment (jgfoster) [inline src/UIState/MainMenu.h:35]: alphabetize these

File: src/UIState/SetKD.cpp [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/**
+ * SetKD.cpp
+ */
+
+#include "SetKD.h"
+
+#include "../Devices/EEPROM_TC.h"
```
Comment (jgfoster) [inline src/UIState/SetKD.cpp:7]: See if it compiles without the "../" (see, e.g., scr/UIState/MainMenu.cpp:4).

**Review by jgfoster — CHANGES_REQUESTED** [review]
This is nice. I've requested a couple small changes.

**Review by prestoncarman — APPROVED** [review]
LGTM after you fixed @jgfoster comments.

File: src/UIState/MainMenu.h (hunk 1 above)
Comment (jgfoster) [reply src/UIState/MainMenu.h:35]: Actually, no, we want the menus to show in a priority or "natural" order.

**Pull request merged**